frontend/simple/components/GroupsMinIncome.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <section class="has-text-right">
     <p class="min-income-label">Min Income</p>
-    <p class="min-income">${{ group.incomeProvided}}</p>
+    <p class="min-income"><cyy>{{ group.incomeCurrency }}</cyy>{{ group.incomeProvided }}</p>
```

Comment:
I have a few concerns about this `cyy` component:

1. It doesn't really do much, so it seems unnecessary
2. Because it doesn't use bindings, it's not useful in situations where the input currency changes
3. As far as I can tell, it would probably be much simpler to just use a computed property instead of a `<cyy>` component